packages/react-components/react-table/src/components/TableRow/useTableRowStyles.ts

```diff
@@ -14,21 +14,34 @@ export const tableRowClassNames: SlotClassNames<TableRowSlots> = {
 const useStyles = makeStyles({
   root: {
     display: 'flex',
-    minHeight: '44px',
     color: tokens.colorNeutralForeground1,
     ':hover': {
       backgroundColor: tokens.colorNeutralBackground1Hover,
     },
+  },
+
+  medium: {
+    minHeight: '44px',
     ...shorthands.borderBottom('1px', 'solid', tokens.colorNeutralStroke2),
   },
+
+  small: {
+    minHeight: '34px',
+    ...shorthands.borderBottom('1px', 'solid', tokens.colorNeutralStroke2),
+  },
+
+  smaller: {
+    minHeight: '24px',
+    fontSize: '12px',
```

Comment:
No tokens for font-size?

Reply:
ah yeah, good point, I'll hunt them down

Reply:
I've replaced all spacing/font values with tokens. The row sizes don't exist as tokens